x-pack/plugins/security_solution/public/common/components/event_details/columns.tsx

```diff
@@ -14,7 +14,8 @@ import { BrowserFields } from '../../containers/source';
 import { OnUpdateColumns } from '../../../timelines/components/timeline/events';
 import * as i18n from './translations';
 import { EventFieldsData } from './types';
-import { BrowserField, ColumnHeaderOptions } from '../../../../common';
+import { ColumnHeaderOptions } from '../../../../common/types';
```

Comment:
Same here, can these be `import type`?

Reply:
These can, but most of the times these are auto-inferred by typescript. There are linter rules for forcing these with an auto-fix as well so I am not going to overly worry about them in these few cases for this PR.